src/ucp/wireup/select.c

```diff
@@ -1764,13 +1784,19 @@ ucp_wireup_search_lanes(const ucp_wireup_select_params_t *select_params,
         return status;
     }
 
-    status = ucp_wireup_add_rma_bw_lanes(select_params, select_ctx);
+    status = ucp_wireup_add_tag_lane(select_params, &am_info, err_mode,
+                                     select_ctx);
     if (status != UCS_OK) {
         return status;
     }
 
-    status = ucp_wireup_add_tag_lane(select_params, &am_info, err_mode,
-                                     select_ctx);
+    /* Add slow protocols on the remaining lanes */
+    status = ucp_wireup_add_cm_lane(select_params, select_ctx);
```

Comment:
can CM lane be added last?

Reply:
can explain?

Reply:
move after `ucp_wireup_add_rma_bw_lanes` call

Reply:
CM lane is a pure control path lane, rma_bw is a data path. So, ordering would be:
1) fast data path lanes
2) slow data path lanes
3) control path lane(s)